{{ cookiecutter.repo_name }}/docs/structure/README.md

```diff
@@ -152,6 +127,22 @@ An overview of the Git repository, including all necessary instructions to execu
 
 A list of Python package requirements for this Git repository, which can be installed using the `pip install` command.
 
-```
+```shell
 pip install --requirement requirements.txt
 ```
+
+Alternatively, to install the requirements file along with pre-commit hooks, run the following command:
+
+```shell
+make requirements
+```
+
+[code-of-conduct]:../contributor_guide/CODE_OF_CONDUCT.md
+[contributing]: ../contributor_guide/CONTRIBUTING.md
+[detect-secrets]: https://github.com/Yelp/detect-secrets
+[direnv]: https://direnv.net/
+[docs-pre-commit-hooks]: ../contributor_guide/pre_commit_hooks.md
+[docs-updating-gitignore]: ../contributor_guide/updating_gitignore.md
+[flake8]: https://gitlab.com/pycqa/flake8
+[gds-way]: https://gds-way.cloudapps.digital
+[gds-way-flake8]: https://gds-way.cloudapps.digital/manuals/programming-languages/python/python.html#common-configuration
```

Comment:
```suggestion
[code-of-conduct]:../contributor_guide/CODE_OF_CONDUCT.md
[contributing]: ../contributor_guide/CONTRIBUTING.md
[coverage]: https://coverage.readthedocs.io/en/coverage-5.3/
[detect-secrets]: https://github.com/Yelp/detect-secrets
[direnv]: https://direnv.net/
[docs-pre-commit-hooks]: ../contributor_guide/pre_commit_hooks.md
[docs-updating-gitignore]: ../contributor_guide/updating_gitignore.md
[flake8]: https://gitlab.com/pycqa/flake8
[gds-way]: https://gds-way.cloudapps.digital
[gds-way-flake8]: https://gds-way.cloudapps.digital/manuals/programming-languages/python/python.html#common-configuration
[pytest]: https://docs.pytest.org/en/stable/
```
`coverage` and `pytest`  were missing from the list and so not rendering as url in the text,

Reply:
Made a similar change, but dropped the coverage version within its hyperlink. Changes made in commit [`212b899`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/212b899c7865d546ac3dcb936a39292687f4a850). Marking as **Resolved**.